client/web/src/repo/tree/ViewGrid.tsx

```diff
@@ -56,6 +59,34 @@ const viewsToReactGridLayouts = (views: ViewProviderResult[]): ReactGridLayouts
     return reactGridLayouts
 }
 
+interface InterInsightDescriptionProps {
+    title: string
+    icon: MdiReactIconComponentType
+}
+
+const stopPropagation: React.MouseEventHandler<HTMLElement> = event => event.stopPropagation()
+
+const InsightDescription: React.FunctionComponent<InterInsightDescriptionProps> = props => {
+    const { icon: Icon, title } = props
+
+    return (
+        <small title={title} className="view-grid__view-id text-muted" onMouseDown={stopPropagation}>
+            <Icon className="icon-inline view-grid__view-id-icon" />
```

Comment:
Trailing whitespace in TSX gets removed, so if you need a space before a line break, you need to add it like this. Prettier will also add this if it automatically breaks up a long line. Then you don't need the class for margin.
```suggestion
            <Icon className="icon-inline" />{' '}
```

Reply:
I've removed class and after Prettier step we have something like
<img width="712" alt="image" src="https://user-images.githubusercontent.com/18492575/112447896-e13b3f80-8d62-11eb-8ffc-b45cd397ab3c.png">

But that works as expected

Reply:
Yeah, because then it's not trailing whitespace